portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,12 +39,14 @@
 import com.google.appengine.api.images.ImagesService;
 import com.google.appengine.api.images.ImagesServiceFactory;
 import com.google.appengine.api.images.ServingUrlOptions;
-import java.io.IOException;
 import java.io.PrintWriter;
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.util.Map;
 import java.util.Optional;
+import com.google.cloud.language.v1.Document;
```

Comment:
The imports should be alphabetized.

Reply:
Done!

I sorted the imports for the other classes as well.